app/src/components/RobotSettings/__tests__/InformationCard.test.js

```diff
@@ -0,0 +1,196 @@
+// @flow
+// robot information card tests
+import * as React from 'react'
+import { Link } from 'react-router-dom'
+
+import { mountWithStore } from '@opentrons/components/__utils__'
+import { LabeledValue, SecondaryBtn, Tooltip } from '@opentrons/components'
+
+import * as Buildroot from '../../../buildroot'
+import * as Discovery from '../../../discovery'
+import { mockConnectableRobot } from '../../../discovery/__fixtures__'
+import { checkShellUpdate } from '../../../shell'
+import { InformationCard } from '../InformationCard'
+
+import type { State } from '../../../types'
+import type { DiscoveredRobot } from '../../../discovery/types'
+
+jest.mock('react-router-dom', () => ({
+  Link: 'a',
+}))
+
+jest.mock('../../../buildroot/selectors')
+jest.mock('../../../discovery/selectors')
+
+const getBuildrootUpdateDisplayInfo: JestMockFn<
+  [State, string],
+  $Call<typeof Buildroot.getBuildrootUpdateDisplayInfo, State, string>
+> = Buildroot.getBuildrootUpdateDisplayInfo
+
+const getRobotApiVersion: JestMockFn<
+  [DiscoveredRobot],
+  $Call<typeof Discovery.getRobotApiVersion, DiscoveredRobot>
+> = Discovery.getRobotApiVersion
+
+const getRobotFirmwareVersion: JestMockFn<
+  [DiscoveredRobot],
+  $Call<typeof Discovery.getRobotFirmwareVersion, DiscoveredRobot>
+> = Discovery.getRobotFirmwareVersion
+
+const getRobotProtocolApiVersion: JestMockFn<
+  [DiscoveredRobot],
+  $Call<typeof Discovery.getRobotProtocolApiVersion, DiscoveredRobot>
+> = Discovery.getRobotProtocolApiVersion
+
+const MOCK_ROBOT_VERSION = '1.2.3'
+const MOCK_FIRMWARE_VERSION = '4.5.6'
+const MOCK_MIN_PAPI_VERSION = '1.0'
+const MOCK_MAX_PAPI_VERSION = '2.8'
+describe('InformationCard', () => {
+  const render = (robot = mockConnectableRobot) => {
+    const updateUrl = `/robot/${robot.name}/update`
```

Comment:
Does not affect the test, but for accuracy:

```suggestion
    const updateUrl = `/robots/${robot.name}/update`
```